tests/remote-environment-controller-tests/test/controllertests/controller_test.go

```diff
@@ -0,0 +1,107 @@
+package controllertests
+
+import (
+	"github.com/kyma-project/kyma/tests/remote-environment-controller-tests/test/testkit"
+	"github.com/stretchr/testify/require"
+	"k8s.io/apimachinery/pkg/apis/meta/v1"
+	"testing"
+	"time"
+)
+
+const (
+	initialWaitTime = 10
+	retryWaitTime   = 5
+	retryCount      = 6
+)
+
+func TestRemoteEnvironmentCreation(t *testing.T) {
+
+	config, err := testkit.ReadConfig()
+	require.NoError(t, err)
+
+	k8sResourcesClient, err := testkit.NewK8sResourcesClient(config.Namespace)
+	require.NoError(t, err)
+
+	helmClient := testkit.NewHelmClient(config.TillerHost, retryCount, retryWaitTime*time.Second)
+	testReName := "test-create-re"
+	k8sResourcesChecker := testkit.NewK8sResourceChecker(testReName, k8sResourcesClient, retryCount, retryWaitTime*time.Second)
```

Comment:
Please define retryWaitTimeSeconds with the following value: retryWaitTime*time.Second.